src/lib/api/user/server/userApiClient.server.ts

```diff
@@ -20,11 +22,80 @@ interface IUsersResponse {
 export const fetchUserList = async (id: string): Promise<IUser[]> => {
   try {
     const response: IUsersResponse = await request<IUsersResponse>({
-      url: API_BASE_URL + '/groups/' + id + '/users',
+      url: `${API_BASE_URL}/groups/${id}/users`,
       method: 'GET',
```

Comment:
I try not to do any refactoring except for the use of template literal in line 25, to keep the scope narrow

Reply:
I understood :) Thank you for fixing!